Angular-CLI/README.md

```diff
@@ -39,15 +36,15 @@ Then click on the gear icon to configure a launch.json file, selecting **Chrome*
 
    ![add-chrome-debug](https://user-images.githubusercontent.com/2836367/27004175-77582668-4dca-11e7-9ce8-30ef3af64a36.png)
 
-- Replace content of the generated launch.json with the following three configurations:
+- Replace content of the generated launch.json with the following three configurations (use `pwa-msedge` for MS Edge):
```

Comment:
The change to pwa-msedge/chrome isn't required, see https://github.com/microsoft/vscode-js-debug/issues/1065

Reply:
Interesting, I was just changing that based on what VS Code was telling me:
![image](https://user-images.githubusercontent.com/2529744/128381226-00cc5bf3-5be2-4738-a03b-0038a0d084aa.png)

Reply:
The Debugger for Chrome is still needed to use `type: chrome` in the currently-released VS Code versions, but that will no longer be the case in 1.60.0 (which releases to stable early next month, and should arrive in Insiders today)